utils/mysql-queries/insertPublisherPremiumUser.ts

```diff
@@ -0,0 +1,13 @@
+import { botPublisherUserPremiums } from '../../db/schema'
+import { TDbConnection } from '../types'
+
+export const insertPublisherPremiumUser = async (db: TDbConnection, id: number) => {
+  const untilTimestamp = Date.now() / 1000 + 60 * 60 * 24 * 31
```

Comment:
You should allow paying forward. Get current `untilTimestamp` from the database, and increase it. For example, a user may want to pay in advance before their premium is expired.

Reply:
Yes, this was an MVP version. I fixed this in the next PR.